.gitignore

```
@@ -8,6 +8,7 @@ yarn.lock
 target
 maven
 stats.html
+wix-rich-content.iml
```

```suggestion
*.iml
```
and move it down to be under the `#Webstorm` section